src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -302,6 +303,23 @@ export default {
 			}
 		},
 	},
+
+	// TODO: move the implementation to @nextcloud-vue/NcListItem
+	watch: {
+		'item.displayName': {
+			immediate: true,
+			handler(value) {
+				this.$nextTick().then(() => {
+					const titleSpan = this.$refs.listItem?.$el?.querySelector('.line-one__title')
+
```

Comment:
If `$refs.listItem` is always defined, then all optional chainings `?.` are redundant.

But if `$refs.listItem` maybe `undefined`, then it will fail anyway with a call of undefined. In such a case, it should have optional chaining on the function call `querySelector?.('.line-one__title')`.

`<NcListItem>` is always rendered: there is no `v-if` on it or anything. IMO, we may drop checking with optional chaining here.